### ISSUE TITLE
🐛 BUG: Irresponsive shortcut button

### ISSUE THREAD

**Issue by fingjohnwick** [dailydotdev/daily#1269]
### What went wrong? 🤔

In the New Tab page, there is a feature that allows us to add custom shortcuts. After adding a few shortcuts, I tried to edit some of it. When I tried to edit it using the 3 dots button, it is not working. Nothing happens when I click the button.

### Expected Behavior

I expected to see a option for editing the shortcuts.

### Steps to Reproduce Issue

```bash
1. Go to 'New Tab Page'
2. Click on 'the 3 dots next to the shortcut'
3. See the error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Environment

Brave Version 1.68.117 Chromium: 127.0.6533.26 (Official Build) beta (64-bit)

### Browsers

Brave

### OS

Windows

### Version of daily.dev

3.33.1

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by capJavert** [comment]
Hello, this was just introduced and fixed recently in https://github.com/dailydotdev/apps/pull/3339, so hopefully in next extension version it should work. cc: @rebelchris

**Comment by rebelchris** [comment]
Hi @fingjohnwick thanks for reporting.

We submitted the fix in version 3.33.2 please keep an eye out for that release and thanks for logging the issue.
